GOVERNANCE.md

```diff
@@ -1,132 +1,175 @@
 
-#### Current list of Committers
-The list is available here: https://github.com/orgs/OpenRefine/people.
+**Current list of Core Developers:**
+- Tom Morris [@tfmorris](https://github.com/tfmorris) and on the [forum](https://forum.openrefine.org/u/tfmorris/summary)
+- Antonin Delpeuch until March 2025: [@wetneb](https://github.com/wetneb) and on the [forum](https://forum.openrefine.org/u/antonin_d/summary).
 
-### Release Manager
+#### Release Manager
 The Release Manager is responsible to
 * Coordinate with the community to select which issues and pull requests are part of new releases
 * Prepare and coordinate publishing new releases
 
-The current Release Manager is [Tom Morris](https://github.com/tfmorris).
+The current Release Manager is Tom Morris [@tfmorris](https://github.com/tfmorris) and on the [forum](https://forum.openrefine.org/u/tfmorris/summary).
 
-### Steering Committee
-The steering committee oversees the general direction of the project and initiates connections and collaborations with other organizations and projects.
+### Code of Conduct Committee
+The code of conduct committee is responsible to
+- Respond to [code of conduct](https://github.com/OpenRefine/OpenRefine/blob/master/CODE_OF_CONDUCT.md) report
+- Provide at least a quarterly report on the forum [day-to-day operations](https://forum.openrefine.org/c/community/operations/10)
 
-* Advise the Project’s staff on processes, strategy, and operations;
-* Participate in decision making and/or review of roadmaps, as time allows;
-* Participate to some Steering Committee meetings, when time allows;
-* Help the Project build connections and partnerships by helping project leadership to steward relationships with funders and partners, making strategic introductions for project leadership, reviewing documents when needed, as time allows;
-* Act as an advocate for the Project in events and support the project’s communication online, as time allows.
-
-#### How to be part of the Steering Committee
-Steering Committee members are invited by OpenRefine's Advisory Committee
-
-#### Current list of Steering Committee members
-There is currently no Steering Committee as the project is looking for a better format. See discussion [Fwd: Steering committee being disbanded?](https://forum.openrefine.org/t/fwd-steering-committee-being-disbanded/430/) and [Proposition to create OpenRefine Ambassador Council](https://forum.openrefine.org/t/proposition-to-create-openrefine-ambassador-council/462)
+**How to join the Code of Conduct Committee?**. Committers are voting-binding members; they select and elect new members through a voting process. Nominations should be submitted to the [developer discussion list](https://forum.openrefine.org/c/dev/8), where contributors can nominate themselves or other contributors. See the New Member Election section below for more details.
 
 ### Advisory Committee
```

Comment:
From here on I think the document do no longer describe the governance of the project but instead the relationship between CS&S and the AC. This is clearly not aligned with the Apache Decision Making process nor do it concern the open source project.

I would suggest moving this somewhere else.

Reply:
To me the AC is currently a component of the project, so it makes sense that it's described in this document. I think @magdmartin's intention is to first update the governance document to reflect the reality of the current situation, and currently the AC does feel like a part of the project to me, so I would keep it. The intention is not to set this in stone, but do incremental updates as the subject is contentious.

There could be further changes later on: changing the way the AC works, or deleting it entirely and leaving CS&S to go back to being just a GitHub project. I think it's best to discuss those separately.

Reply:
> and currently the AC does feel like a part of the project to me, so I would keep it

This is probably because you are a part of it? You are the only notable GitHub contributor who are also a part of the AC...